multiview_mapping_toolkit/meshes/meshes.py

```diff
@@ -1767,13 +1768,15 @@ def vis(
 
         # Set the mesh kwargs if not set
         if mesh_kwargs is None:
+            mesh_kwargs = {}
             if self.is_discrete_texture() and len(self.get_label_names()) <= 10:
-                mesh_kwargs = TEN_CLASS_VIS_KWARGS
+                tab10_colors = [matplotlib.colors.to_hex(c) for c in plt.get_cmap('tab10').colors]
```

Comment:
Minor option to potentially simplify this. Rather than having the `tab10/tab20` colors, you could just set one variable based on the condition. Then you wouldn't need to duplicate the following two lines.